### PR TITLE
Dw 2689 and dw2690

### PR DESCRIPTION
**Pull request by John-Milne**
Use this branch for both DW-2689 and DW-2690

### CODE REVIEW

**Review by GeoffreyHayward — CHANGES_REQUESTED** [review]
This PR has noise that is unrelated to your change. Please rebase it and double-check that it only contains your expected changes and files.

**Comment by John-Milne** [comment]
Updated grid breakout to use the same method as the video Jira DW-2692 so passed back for testing

File: repository-data/webfiles/src/main/resources/site/freemarker/nhsd-common/macro/heroes/base-hero.ftl [hunk 1]
```diff
@@ -126,6 +141,17 @@ default|backgroundImage|image|accentedImage|accentedImageMirrored
...
+                            <figcaption>
+				                <#if options.link?has_content>
+				                    <a class="nhsd-a-link" href="${options.link}" data-uipath="ps.publication.image-section.link">
+				                </#if>
+				                ${htmlCaption?no_esc}
+				                <#if options.link?has_content>
+						            </a>
+						        </#if>
+			                </figcaption>
+			            </#if>
```
Comment (GeoffreyHayward) [inline repository-data/webfiles/src/main/resources/site/freemarker/nhsd-common/macro/heroes/base-hero.ftl:154]: Please tidy the formatting here.

**Review by GeoffreyHayward — CHANGES_REQUESTED** [review]
It's looking good. Please tidy the indentation first. Then it's good to go.

File: repository-data/webfiles/src/main/resources/site/freemarker/nhsd-common/macro/sections/imageModule.ftl [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+		            <div style="padding-right: 10px;"><@hst.html hippohtml=section.text contentRewriter=brContentRewriter/></div>
+	            </div>
+            </#if>
+            <#if fullWidth>
+            <div class="nhsd-t-grid">
+            <div class="nhsd-t-row">
+            <div class="nhsd-t-col-xs-12 nhsd-t-col-s-8">
+            <div>
+        	</#if>
+        </#if>
```
[the remark this answers is not in the document]
Comment (John-Milne) [reply repository-data/webfiles/src/main/resources/site/freemarker/nhsd-common/macro/sections/imageModule.ftl:85]: Formatting looks fine in IntelliJ and if you click on edit in here

[the remark this answers is not in the document]
Comment (John-Milne) [reply repository-data/webfiles/src/main/resources/site/freemarker/nhsd-common/macro/sections/imageModule.ftl]: Inconsistencies show up in vi, so corrected using that

**Pull request merged**